src/main/java/uk/gov/pay/adminusers/service/ServiceRoleUpdater.java

```diff
@@ -64,8 +64,8 @@ public ServiceRoleUpdater(UserDao userDao, ServiceDao serviceDao, RoleDao roleDa
 
         ServiceRoleEntity serviceRoleEntity = servicesRoleMaybe.get();
 
-        if (!roleEntity.getId().equals(ROLE_ADMIN_ID)) {
-            if (serviceDao.countOfRolesForService(serviceId, ROLE_ADMIN_ID) <= adminsPerServiceLimit) {
+        if (!roleEntity.isAdmin()) {
```

Comment:
As discussed, yes is a bit confusing thinking about this situation from a _selfservice_ as client POV but makes sense if the role to changes is something else but Admin since we can't left a service empty of Admin users (at least one must exist)